lib/host-connection-pool.js

```diff
@@ -78,6 +77,19 @@ class HostConnectionPool extends events.EventEmitter {
     });
   }
 
+  getInFlight() {
```

Comment:
I agree that changing this from passively managing a running count to recalculating on every invocation is the right thing to do because:

1. This method isn't called anywhere by the driver itself, so unless someone uses it explicitly, the cost is minimal.  We expect it to be used for host selection (choose least busy host), so it may happen on each request but it's still probably cheaper than managing the running count (since something needed to be incremented/decremented previously to manage it anyways).
2. If you only have 1 connection per host (which should be typical), we don't need to sum things up.